CMakeLists.txt

```diff
@@ -77,20 +77,19 @@ endif (USE_RUNPATH)
 
 if (MSVC)
   add_definitions( "/W3 /D_CRT_SECURE_NO_WARNINGS /wd4996 /wd4244 /wd4267")
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /MP")
+  set(CMAKE_CXX_FLAGS "/MP ${CMAKE_CXX_FLAGS}")
 else()
-  if (NOT DEFINED CMAKE_CXX_FLAGS)
```

Comment:
We should give people the chance to overwrite these flags.

Maybe introducing a CACHE variable OPM_CMAKE_CXX_FLAGS_DEBUG and and adding them to CMAKE_CXX_FLAGS_DEBUG is an option
```
set(OPM_CMAKE_CXX_FLAGS_DEBUG  "-pipe -std=c++0x -pedantic -Wall -Wextra -Wformat-nonliteral -Wcast-align -Wpointer-arith -Wmissing-declarations -Wcast-qual -Wshadow -Wwrite-strings -Wchar-subscripts -Wredundant-decls"
   CACHE STRING "Default flags for OPM")
set(CMAK_CXXFLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} ${OPM_CMAKE_CXX_FLAGS_DEBUG}")
```
IMHO the same should be done for the other flags (RELEASE, DEBUG), too.

By the way, what happens if we rerun CMake? Do they get duplicated? I guess that might have been  a reason for the former  if statement.